frontend/src/components/BookPage.js

```diff
@@ -237,80 +219,60 @@ class BookPage extends Component {
+              <Tooltip placement="right" enterTouchDelay={0} title={t(params.row.type)}>
+                <div style={{ cursor: "pointer" }}>
+                  {amountToString(params.row.amount, params.row.has_range, params.row.min_amount, params.row.max_amount)}
+                </div>
+              </Tooltip>
+            )} 
+          },
           { field: 'currency', headerName: t("Currency"), width: 85,
-          renderCell: (params) => {return (
-            // <Tooltip placement="left" enterTouchDelay={0} title={params.row.payment_method}>
-              <div style={{ cursor: "pointer", display:'flex',alignItems:'center', flexWrap:'wrap'}}>
-                {params.row.currency+" "}
-                <FlagWithProps code={params.row.currency} />
-              </div>
-            // </Tooltip>
-          )} },
+            renderCell: (params) => { 
+              const currencyCode = this.getCurrencyCode(params.row.currency)
+              return (
+                <div style={{ cursor: "pointer", display:'flex',alignItems:'center', flexWrap:'wrap'}}>
+                  {currencyCode + " "}
+                  <FlagWithProps code={currencyCode} />
+                </div>
+              )
+            } 
+          },
           { field: 'payment_method', headerName: t("Payment Method"), width: 180, hide:'true'},
           { field: 'payment_icons', headerName: t("Pay"), width: 75 ,
-          renderCell: (params) => {return (
-            <div style={{position:'relative', left:'-4px', cursor: "pointer", align:"center"}}><PaymentText othersText={t("Others")} size={16} text={params.row.payment_method}/></div>
-          )} },
+            renderCell: (params) => {return (
+              <div style={{position:'relative', left:'-4px', cursor: "pointer", align:"center"}}><PaymentText othersText={t("Others")} size={16} text={params.row.payment_method}/></div>
+            )} 
+          },
           { field: 'price', headerName: t("Price"), type: 'number', width: 140, hide:'true',
-          renderCell: (params) => {return (
-            <div style={{ cursor: "pointer" }}>{pn(params.row.price) + " " +params.row.currency+ "/BTC" }</div>
-          )} },
+            renderCell: (params) => {return (
+              <div style={{ cursor: "pointer" }}>{pn(params.row.price) + " " + params.row.currency+ "/BTC" }</div>
+            )} 
+          },
           { field: 'premium', headerName: t("Premium"), type: 'number', width: 85,
             renderCell: (params) => {return (
-              <Tooltip placement="left" enterTouchDelay={0} title={pn(params.row.price) + " " +params.row.currency+ "/BTC" }>
+              <Tooltip placement="left" enterTouchDelay={0} title={pn(params.row.price) + " " + params.row.currency + "/BTC" }>
               <div style={{ cursor: "pointer" }}>{parseFloat(parseFloat(params.row.premium).toFixed(4))+"%" }</div>
               </Tooltip>
-            )} },
-          ]}
+            )} 
+          }]}
```

Comment:
In another iteration I see this array reusable for both tables mobile and desktop